### PR TITLE
2019-05-06 Release "Operation previews thunder"

### PR DESCRIPTION
**Pull request by jnoller**

### CODE REVIEW

File: previews.md [hunk 1]
```diff
@@ -155,3 +206,7 @@ https://github.com/Azure/application-gateway-kubernetes-ingress
 [5]: https://docs.microsoft.com/en-us/azure/aks/cluster-autoscaler
 [6]: https://docs.microsoft.com/en-us/azure/virtual-machine-scale-sets/overview
 [7]: https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest
+
+[api server]: https://docs.microsoft.com/en-us/azure/aks/api-server-authorized-ip-ranges
+[windows]: https://docs.microsoft.com/en-us/azure/aks/windows-container-cli
```
Comment (MikkelHegn) [inline previews.md]: These docs are not public yet. Don't know when they will be...

Comment (jnoller) [reply previews.md]: Woot, removed.

File: previews.md [hunk 2]
```diff
@@ -44,6 +44,57 @@ az extension remove --name aks-preview
...
+
+Then refresh your registration of the AKS resource provider:
+
+```
+az provider register -n Microsoft.ContainerService
+```
+
+### Multiple Node Pools
+
+Multiple Node Pool support is now in public preview. You can see the
```
Comment (tomkerkhove) [inline previews.md]: 🙌

**Comment by seanmck** [comment]
LGTM

**Comment by victornoel** [comment]
@jnoller @seanmck any way to get aks-engine 0.33.6 included in this?
See #957 as a rational. It's only a patch version.
Thanks :)

**Comment by jnoller** [comment]
@victornoel Not at this time, the new AKS-engine version is prioritized for the next release.

**Comment by victornoel** [comment]
@jnoller thanks, that's good news :)

**Pull request merged**